core/arch/riscv/kernel/entry.S

```diff
@@ -71,9 +71,21 @@
 .endm
 
 .macro set_satp
+	/*
+	 * a0 = hartid
+	 * a1 = address of boot_mmu_config.satp[0]
+	 * a2 = size of CSR SATP
+	 *
+	 * This hart's SATP is value of (a1 + (a0 * a2)).
```

Comment:
"This hart's SATP is **of** value **~~of~~** (a1 + (a0 * a2))."

Reply:
Fixed in https://github.com/OP-TEE/optee_os/pull/6713/commits/22f2fd2636e42d119e0fdd39a61ccf677b4fc73c